### ISSUE TITLE
[BUG]: `foodemo` sounds funny in portuguese

### ISSUE THREAD

**Issue by brunolnetto** [thieu1995/foodemo#1]
### Description of the bug

`foodemo` means `we fucked up` in portuguese. I suggest to change its name. :-P

### Steps To Reproduce

1. Go to Settings;
2. Change library name to something nicer then `foodemo`.

**Comment by thieu1995** [comment]
@brunolnetto ,

I'm just trying to learn somthing new from this guy 
https://www.youtube.com/watch?v=YXcXIr835mw&t=53s

**Comment by brunolnetto** [comment]
You can copy-cat my librariess structure: 

https://pypi.org/project/eule/
https://pypi.org/project/pybary/

**Comment by brunolnetto** [comment]
To compliment your effort would be weird, since you already know it is great. But take a look at the above libraries I mention. I made an active effort to implement the package manager [poetry](https://python-poetry.org/).

**Comment by thieu1995** [comment]
@brunolnetto ,

Thank you so much. But I really have no idea how to config them from scratch. I don't understand these commands and configurations. Do you make tutorial video for them

**Comment by brunolnetto** [comment]
Sure, but I am a bit lazy. :-P Follow me:

1. Old way to release use separate commands to generate a local virtual environment, test and add plugins. Poetry makes it all in one in a file with extension [pyproject.toml](https://github.com/asmove/pybary/blob/main/pyproject.toml) ;
2. [MakeFile](https://github.com/asmove/pybary/blob/main/Makefile) makes this process easierer through command `make` followed by some aliases;

Like on javascript, poetry generates a `lock` file to build a all-in-one package file for the sake of performance. There are [plenty of commands](https://python-poetry.org/docs/cli/), which I am not competent enough to summarize. 

In addition, we can setup [pre-commit](https://github.com/asmove/pybary/blob/main/.pre-commit-config.yaml) on their website by sign up of repository of desire and configuration file on repository root [likeso](https://github.com/asmove/pybary/blob/main/.pre-commit-config.yaml).

**Comment by thieu1995** [comment]
That sound a lots of works to do, and that is a problem. I don't have enough time to learn these commands (not right now). Anyway, thank you for trying to help.

**Comment by brunolnetto** [comment]
Yes, a bit a learning on the way. The triumph on such new learning is to unite separate concepts together.
